homeassistant/components/zha/core/registries.py

```diff
@@ -151,11 +151,11 @@ def weight(self) -> int:
 
         Most specific matches should be preferred over less specific. Model matching
         rules have a priority over manufacturer matching rules and rules matching a
-        single model/manufacturer get a better priority over rules matching multiple
+        single model/manufacturer get a better priority overrules matching multiple
         models/manufacturers. And any model or manufacturers matching rules get better
-        priority over rules matching only channels.
+        priority overrules matching only channels.
         But in case of a channel name/channel id matching, we give rules matching
-        multiple channels a better priority over rules matching a single channel.
+        multiple channels a better priority overrules matching a single channel.
```

Comment:
These fixes are incorrect. This is talking about one set of rules A (noun) applying over another set of rules B (also noun). Overrules is a verb, which could be used such as "rules A overrules rules B", but "rules A overrules" doesn't make sense. Overruling could be used instead, though it should still be in explicit reference to rules B, so I think the original is clearest.

Reply:
Yep, you're absolutely right! Not sure how I missed this

Reply:
Fixed: https://github.com/home-assistant/core/pull/84881/commits/627a8efa1a865ed7dc7f6b76dfe9db145862b541